### DISCUSSION TITLE
I keep getting this error when i after using sergen g to import data table. Only thing i have changed is some of the field names because they had # or % in them.

### DISCUSSION THREAD

**Discussion by thebom999** [serenity-is/Serenity#6560] [Help]
![image](https://user-images.githubusercontent.com/33629052/195609644-1263e4e9-b921-42c6-b761-83f9067c093a.png)

**Comment by edwardch** [comment]
Can you paste your code for this class?

BTW you can format your code using:

` ```cs `
`// your code here`
` ``` `
